lib/ansible/modules/source_control/gitlab_hook.py

```diff
@@ -309,6 +316,7 @@ def main():
         project=dict(type='str', required=True),
         hook_url=dict(type='str', required=True),
         push_events=dict(type='bool', default=True),
+        push_events_branch_filter=dict(type='str'),
```

Comment:
You should maybe set the default value to `""`

Reply:
Thanks for your remark, without the default value the module was not doing field's cleaning if the parameter was not present. Now, with it, it does